.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
If flake is used, the configuration file should have to same rules as pylint.

Reply:
I'll remove it. Was checked unintended.